release notes/v1.0.0-rc1.md

```diff
@@ -0,0 +1,186 @@
+k6 `v1.0.0-rc1` is here 🎉! This release includes:
+
+- A preview of the next `v1.0.0` release
+- `k6/experimental/webcrypto` promoted to stable and available globally under `crypto`.
+- A revamped end-of-test summary aiming to bring an easier way to understand test results.
+
+## Breaking changes
+
+- [#4541](https://github.com/grafana/k6/pull/4541) Commas(`,`) are now supported in the values of `--tag` cli flag. This is a breaking change as previously a comma meant the start of a new set of tag-values. As comma is a valid symbol for the value of a tag this is necessary to have equivalency between different ways of setting tags. This still allows multiple tags to be set on the cli with multiple `--tag key=value` arguments.
```

Comment:
@mstoykov It would be good to update the docs for clarifying it

Reply:
Looking at the [doc](https://grafana.com/docs/k6/latest/using-k6/k6-options/reference/#tags) We have never said that it is possible to use `,` to give multiple values. I also looked at taht to see if we actually have ever showed that - and no.

Reply:
I know, and this is the point. I think we should improve it and clarify the case. We should move the various examples / documented cases to show how to use multiple tags, instead of all them on the case with only one tag.

Reply:
https://github.com/grafana/k6-docs/pull/1909